### DISCUSSION TITLE
About the CORS and cookie problem

### DISCUSSION THREAD

**Discussion by bentshrimp** [AgainIoT/Open-Set-Go#32] [General]
# About the CORS problem
The client, ReactJS app and the server, NestJS app are being run on a same machine, AWS EC2. This is the only reason why we don't encounter CORS error. By the way, it happens when client and server are being run on different machines.
# About the cookie problem
When the client and server are running on a same machine, browser dev tool doesn't show the cookie. On the other hand, when they are not, even without getting cookie and writing cookie, browser says it has the cookie. I have no idea what happens here.

We should keep trying to figure out how to fix these problem.

**Comment by ymw0407** [comment]
lets resolve this issue at #45
